Makefile

```diff
@@ -307,6 +307,18 @@ test-install-cni: image k8s-install/scripts/install_cni.test
 ## Run what CI runs
 ci: clean static-checks test-cni-versions image test-install-cni
 
+## Deploys images to registry
```

Comment:
@deitch I think we're going with these big block headings now - could you add one here for CD (I believe this is the convention @tomdee has been adding throughout)

Reply:
Sure. The double- comment `##` causes it to show up when you do `make help` (from [this line](https://github.com/deitch/calico-cni-plugin/blob/96659bfc872f2e38a32b9cf50af826aa347008f0/Makefile#L424) ) and happy to add the block heading too. Coming in one minute.

Reply:
Wait a minute, there is one here. See [this](https://github.com/deitch/calico-cni-plugin/blob/96659bfc872f2e38a32b9cf50af826aa347008f0/Makefile#L303-L305) . Is this not good enough?

Reply:
Yep, I've just been adding /CD to that block heading (https://github.com/projectcalico/confd/blob/master/Makefile#L295) so lets go with that

Reply:
Done